.gitignore

```diff
@@ -121,3 +121,6 @@ package-lock.json
 
```

Comment:
As discussed yesterday moving to draft and assigning to myself to take a look in the future.